### PR TITLE
fix:fix key props issues in the blogPosts and features components

### LINKED ISSUE #26: fix: Add key for lists to fix the unique key prop issue.
image

Currently, `warning: Each child in a list should have a unique "key" prop.` is displaying in the console.
This is because some of lists are rendering without any key prop.
Add the unique prop for them.

# Acceptance Criteria:
1. Fix `warning: Each child in a list should have a unique "key" prop.` issue.

### PR DESCRIPTION
**Pull request by Meldy2503**

### CODE REVIEW

**Comment by Meldy2503** [comment]
hello @Min-Ho-Lim, I'm Emelder, I contributed and created a pull request on the issue "key props issues on the features and blogpost component", please kindly review

File: readme.md [hunk 1]
```diff
@@ -1 +1,67 @@
+# Gym Landing Page
```
Comment (Min-Ho-Lim) [inline readme.md]: I am not sure why do you create new `readme.md`?

Comment (Min-Ho-Lim) [reply readme.md]: - [x] Delete `readme.md` from your changes. We already have `README.md`.

**Review by Min-Ho-Lim — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (Meldy2503) [reply readme.md]: @Min-Ho-Lim ok

**Review by Min-Ho-Lim — APPROVED** [review]
Thanks for the contribution. Approved.

**Comment by Meldy2503** [comment]
You're most welcome Sir, thanks for the opportunity

**Pull request merged**